gulpfile.js

```diff
@@ -0,0 +1,39 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var rename = require('gulp-rename');
+var cleanCSS = require('gulp-clean-css');
+var autoprefixer = require('gulp-autoprefixer');
+var browserSync = require('browser-sync').create();
+
+gulp.task('default', ['serve']);
```

Comment:
Why is `serve` the default task? IMO the default should be `minify-css`. Or if you think serve should be default, you should update the README and explain how to only build, not run the server.